K8s-deployment/Charts/auditing-server/README.md

```diff
@@ -1,3 +1,5 @@
+[![Kubescape Status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2FKubescape%2Fjob%2Fkubescape-auditing%2F&label=Kubescape)](https://jenkins.iudx.io/view/Kubescape/job/kubescape-auditing/Kubescape_20Scan_20Report_20for_20Auditing_20Server/)
```

Comment:
```suggestion
[![Kubescape Status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fjob%2Fkubescape-auditing%2F&label=Kubescape)](https://jenkins.iudx.io/job/kubescape-auditing/lastBuild/Kubescape_20Scan_20Report_20for_20Auditing_20Server/)
```

Unnecessary failure points with having the view in the link, it'll fail if the view is ever deleted or even renamed. Remove from both the job URL in the badge and links to reports. And keep report links through the `lastBuild`, this way the report directly takes us to the associated build.